docs/reference/commandline/network_connect.md

```diff
@@ -61,11 +61,8 @@ being connected to.
 ```bash
 $ docker network connect --alias db --alias mysql multi-host-network container2
 ```
-
-You can pause, restart, and stop containers that are connected to a network.
-Paused containers remain connected and can be revealed by a `network inspect`.
-When the container is stopped, it does not appear on the network until you restart
-it.
+A container that is connected to a network does not appear on the
```

Comment:
You can pause, restart, and stop containers that are connected to a network.		 
A container connects to its configured networks when it runs.

Does it look good ?